README.md

```diff
@@ -1,2 +1,2 @@
-# NetTopologySuite.IO.SqlServer
-A Server IO module for NTS that targets .NET Standard.
+# NetTopologySuite.IO.SqlServerBytes
+A SQL Server IO module for NTS which works directly with the serialization format
```

Comment:
@FObermaier Do you want to rename the repo and add this description?

Reply:
👍 Just did that.